### PR TITLE
Add new category for programming language

### PR DESCRIPTION
**Pull request by jaqsparow**

### CODE REVIEW

File: qual_id/pattern.py [hunk 1]
```diff
@@ -46,6 +48,7 @@ class Pattern:
         "instrument": Instrument(),
         "language": Language(),
         "music": Music(),
+        "operatingsystem": OperatingSystem(),
         "planet": Planet(),
```
Comment (gabrielbarker) [inline qual_id/pattern.py:52]: Are you forgetting to add "programminglanguage" to the map?

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

File: qual_id/pattern.py (hunk 1 above)
Comment (jaqsparow) [reply qual_id/pattern.py:52]: oops 😉

**Review by gabrielbarker — APPROVED** [review]
Great contribution @jaqsparow 🙂

**Comment by gabrielbarker** [comment]
Thanks again @jaqsparow !! 🎉

**Pull request merged**